devito/compiler.py

```diff
@@ -271,6 +272,7 @@ def jit_compile(self, soname, code):
             try:
                 with open(src_file, 'r') as f:
                     code = f.read()
+                    code = ''.join([code, '/* Backdoor edit at %s*/ \n' % time.ctime()])
```

Comment:
OK so this way it's 100% sure that each time a different string is compiled right? 

LGTM. Have you tested it?

Reply:
So yes, this is tested. If anybody else would like to try, please go on.